### PR TITLE
Small fixes for runner/utils and code owners update

### PR DESCRIPTION
**Pull request by Alexsandruss**
- Fix for dummy run work: `dataset_path` is referenced and not assigned if dummy run is used
- Error type addition in SW configuration: if `conda list` output is in wrong format (for example, if conda env is broken), TypeError might be raised
- pd.DataFrame.drop calls only with keywords to support future version of Pandas
- Codeowners update
- Update python version
- Fix sklearn version to prevent fails without fixed sklearnex release

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #owners and reviewers
-cuml_bench/*          @Pahandrovich @Alexsandruss @masdevas
-daal4py_bench/*       @Pahandrovich @Alexsandruss @agorshk
-datasets/*            @Alexsandruss @itearsl @masdevas
-modelbuilders_bench/* @Pahandrovich @agorshk
-report_generator/*    @Pahandrovich @Alexsandruss
-sklearn_bench/*       @Pahandrovich @Alexsandruss @agorshk
```
Comment (samir-nasibli) [inline .github/CODEOWNERS:7]: samir-nasibli for sklearn_bench

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,9 +1,9 @@
 #owners and reviewers
-cuml_bench/*          @Pahandrovich @Alexsandruss @masdevas
-daal4py_bench/*       @Pahandrovich @Alexsandruss @agorshk
```
Comment (samir-nasibli) [inline .github/CODEOWNERS:3]: samir-nasibli daal4py_bench

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-xgboost_bench/*       @Pahandrovich @agorshk
-*.md                  @outoftardis
+cuml_bench/*          @Alexsandruss
+daal4py_bench/*       @Alexsandruss @samir-nasibli
+datasets/*            @Alexsandruss
+modelbuilders_bench/* @Alexsandruss
+report_generator/*    @Alexsandruss
+sklearn_bench/*       @Alexsandruss @samir-nasibli
+xgboost_bench/*       @Alexsandruss
+*.md                  @Alexsandruss @maria-Petrova
```
Comment (samir-nasibli) [inline .github/CODEOWNERS:9]: It says that it is unknown codeowner. Could you please check it again why it shows error

**Review by samir-nasibli — APPROVED** [review]

File: .github/CODEOWNERS (hunk 3 above)
Comment (Alexsandruss) [reply .github/CODEOWNERS:9]: @maria-Petrova is not associated with this repo yet.

**Pull request merged**